index.html

```diff
@@ -202,6 +202,23 @@ <h3>Open Demographics</h3>
 					<p>Open Demographics is a recommended set of questions that anyone can use to ask community members about their demographics.</p>
 				</a>
 			</div>
+			<div class="row">
```

Comment:
I moved this whole div up to the projects section, previously you had it in spaces